src/System.Reflection.Emit.ILGeneration/tests/CustomAttributeBuilder/CustomAttributeBuilderCtor2.cs

```diff
@@ -2,984 +2,194 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 // See the LICENSE file in the project root for more information.
 
+using System.Collections.Generic;
 using System.Linq;
 using Xunit;
 
-namespace System.Reflection.Emit.ILGeneration.Tests
+namespace System.Reflection.Emit.Tests
 {
-    public class CustomAttributeBuilderTest : Attribute
-    {
-        public string TestString
-        {
-            get
-            {
-                return TestStringField;
-            }
-            set
-            {
-                TestStringField = value;
-            }
-        }
-
-        public int TestInt32
-        {
-            get
-            {
-                return TestInt;
-            }
-            set
-            {
-                TestInt = value;
-            }
-        }
-
-        public string GetOnlyString
-        {
-            get
-            {
-                return GetString;
-            }
-        }
-
-        public int GetOnlyInt32
-        {
-            get
-            {
-                return GetInt;
-            }
-        }
-
-        public string TestStringField;
-        public int TestInt;
-        public string GetString;
-        public int GetInt;
-
-        public CustomAttributeBuilderTest()
-        {
-        }
-
-        public CustomAttributeBuilderTest(string getOnlyString, int getOnlyInt32)
-        {
-            GetString = getOnlyString;
-            this.GetInt = getOnlyInt32;
-        }
-
-        public CustomAttributeBuilderTest(string testString, int testInt32, string getOnlyString, int getOnlyInt32)
-        {
-            this.TestStringField = testString;
-            this.TestInt = testInt32;
-            this.GetString = getOnlyString;
-            this.GetInt = getOnlyInt32;
-        }
-    }
+    using static Helpers;
```

Comment:
I'd prefer we not use this.  These tests are sprawling, with lots of test class and lots of files, and seeing an unadorned method call suggests that the method is in that test class or a base or something.  It doesn't save much typing, and without it it's very clear where those helpers are coming from.